src/modules/components/proposal/proposalVoting/proposalVoting.stories.tsx

```diff
@@ -409,4 +409,69 @@ export const MultiStageMultiBody: Story = {
     },
 };
 
+export const MultiBodyWithExternalBody: Story = {
+    args: {
+        className: 'max-w-[560px]',
+    },
+    render: (args) => {
+        const bodyList = ['Token holder voting', 'safe.hyper-protocol.eth'];
+        const [tokenSearch, setTokenSearch] = useState<string | undefined>('');
+        const [multisigSearch, setMultisigSearch] = useState<string | undefined>('');
+        const safeExample = {
+            logo: 'https://app.safe.global/images/safe-logo-green.png',
+            label: 'Safe{Wallet}',
+        };
+
+        const minApprovals = 5;
+
+        return (
+            <ProposalVoting.Container {...args}>
+                <ProposalVoting.Stage
+                    name="Token holder voting"
+                    status={ProposalVotingStatus.ACTIVE}
+                    startDate={DateTime.now().toMillis()}
+                    endDate={DateTime.now().plus({ days: 5 }).toMillis()}
+                    bodyList={bodyList}
+                >
+                    <ProposalVoting.BodySummary>
+                        <ProposalVoting.BodySummaryList>
+                            {bodyList.map((bodyId) => (
+                                <ProposalVoting.BodySummaryListItem
+                                    key={bodyId}
+                                    id={bodyId}
+                                    brandedExternal={bodyId === 'safe.hyper-protocol.eth' ? safeExample : undefined}
+                                >
+                                    {bodyId}
+                                </ProposalVoting.BodySummaryListItem>
+                            ))}
+                        </ProposalVoting.BodySummaryList>
+                        <p className="text-center text-neutral-500 md:text-right">
+                            <span className="text-neutral-800">1 body</span> required to approve
+                        </p>
+                    </ProposalVoting.BodySummary>
+                    <ProposalVoting.BodyContent
+                        name="Token holder voting"
+                        status={ProposalVotingStatus.ACTIVE}
+                        bodyId="Token holder voting"
+                    >
+                        <TokenVotingContent tokenSearch={tokenSearch} setTokenSearch={setTokenSearch} />
+                    </ProposalVoting.BodyContent>
+                    <ProposalVoting.BodyContent
+                        name="safe.hyper-protocol.eth"
+                        status={ProposalVotingStatus.ACTIVE}
+                        bodyId="safe.hyper-protocol.eth"
+                        brandedExternal={safeExample}
```

Comment:
Let's hide Votes tab in the content for external body, to showcase how it should be.

```suggestion
                        brandedExternal={safeExample}
                        hideTabs={[ProposalVotingTab.VOTES]}
```